clash-cores/src/Clash/Cores/Xilinx/Ethernet/Gmii.hs

```diff
@@ -0,0 +1,117 @@
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ViewPatterns #-}
+
+{- |
+Copyright   :  (C) 2024, Google LLC
+License     :  BSD2 (see the file LICENSE)
+Maintainer  :  QBayLogic B.V. <devops@qbaylogic.com>
+
+Contains a wrapper for the Xilinx GMII to SGMII PMA core :
+ "LogiCORE IP Ethernet 1000BASE-X PCS/PMA or SGMII"
+ https://www.xilinx.com/xsw/gig_ethernet_pcs_pma
+
+Currently only the Verilog primitive is supported.
+-}
+module Clash.Cores.Xilinx.Ethernet.Gmii (
+  --- Types
+  AutoNegConfig (..),
+  BridgeOutput (..),
+  Config (..),
+  DuplexMode (..),
+  Gmii (..),
+  LinkSpeed (..),
+  Lvds (..),
+  Nanoseconds,
+  Pause (..),
+  Picoseconds,
+  Status (..),
+  --- Gmii to Sgmii bridge
+  gmiiSgmiiBridge,
+)
+where
+
+import Clash.Explicit.Prelude
+
+import Clash.Cores.Xilinx.Ethernet.Gmii.Internal
+import Clash.Signal.Internal (DiffClock (..))
+
+-- | A subset of the relevant output signals of the GMII to SGMII bridge
+data BridgeOutput gmii125 sgmii625 = BridgeOutput
+  { bridgeLvdsOut :: Signal sgmii625 Lvds
+  , bridgeGmiiRx :: Signal gmii125 Gmii
+  , bridgeClk125 :: Clock gmii125
+  , bridgeRst125 :: Reset gmii125
+  , bridgeStatus :: Signal gmii125 Status
+  }
```

Comment:
```suggestion
-- | A subset of the relevant output signals of the GMII to SGMII bridge
data BridgeOutput gmii125 sgmii625 = BridgeOutput
  { bridgeLvdsOut :: Signal sgmii625 Lvds
    -- ^ LVDS output to the PHY
  , bridgeGmiiRx :: Signal gmii125 Gmii
    -- ^ GMII output to the MAC
  , bridgeClk125 :: Clock gmii125
    -- ^ Clock output for the 125 MHz domain
  , bridgeRst125 :: Reset gmii125
    -- ^ Active high reset output for the 125 MHz domain
  , bridgeStatus :: Signal gmii125 Status
    -- ^ Status vector
  }
```

I just copied these descriptions from the `Internal` module; it seems like they are also useful to users of the public API.